src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -87,7 +88,7 @@ const styles = StyleSheet.create({
     fontStyle: 'normal',
     fontWeight: '500',
     fontSize: 14,
-    color: '#000000',
+    color: COLORS.black,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Nice work on replacing the hardcoded color value with a constant from a theme file. This improves maintainability and consistency across the codebase.
```diff
- color: '#000000',
+ color: COLORS.black,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->